docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -34,15 +42,22 @@ PUT /_snapshot/my_repository
 [[slm-gs-create-policy]]
 ==== Setting up a snapshot policy
 
-Now that we have a repository in place, we can create a policy to automatically
-take snapshots. Policies are written in JSON and will define when to take
-snapshots, what the snapshots should be named, and which indices should be
-included, among other things.  We'll use the <<slm-api-put-policy>> API
-to create the policy.
+Once you have a repository in place, you can define an {slm-init} policy to automate your backups. 
+The policy defines when to take snapshots, which indices should be included, 
+and what to name the snapshots. 
+A policy can also specify a <<slm-retention,retention policy>> and 
+automatically delete snapshots when they are no longer needed.
+
+TIP: Don't be afraid to configure a policy that takes frequent snapshots.
+Snapshots are incremental and make efficient use of storage.
+
+You can define and manage policies through the {kibana} Management UI, 
+which invokes the {slm-init} put policy API to create policies according to the options you specify.
```

Comment:
Not sure how useful it is to peel back the hood here. I'd just tell people they can use the UI or the API.
Don't think it matters that they know the UI invokes the API.

> You can define and manage policies through the {kibana} Management UI, 
which invokes the {slm-init} put policy API to create policies according to the options you specify.